Tools/Scripts/webkitpy/port/win_unittest.py

```diff
@@ -35,33 +35,35 @@
 from webkitpy.common.system.systemhost_mock import MockSystemHost
 from webkitpy.common.version_name_map import PUBLIC_TABLE, VersionNameMap
 from webkitpy.port import port_testcase
-from webkitpy.port.win import WinPort
+from webkitpy.port.win import WinCairoPort
 from webkitpy.tool.mocktool import MockOptions
 
 
 class WinPortTest(port_testcase.PortTestCase):
     os_name = 'win'
     os_version = Version.from_name('XP')
     port_name = 'win-xp'
-    port_maker = WinPort
+    port_maker = WinCairoPort
 
+    maxDiff = None
```

Comment:
Which code does use `maxDiff` ?

Reply:
It's a `unittest` thing: https://docs.python.org/3/library/unittest.html#unittest.TestCase.maxDiff

But I should've removed it before committing!